CONTRIBUTING.md

```diff
@@ -245,37 +242,4 @@ Releasing a package is a two-step process.
    done manually with `git tag $tagname && git push upstream $tagname` while
    checked out on the commit that updated `version` in `pubspec.yaml`.
 
-We've recently updated
-[flutter_plugin_tools](https://github.com/flutter/plugin_tools) to wrap both of
-those steps into one command to make it a little easier. This new tool is
-experimental. Feel free to fall back on manually running `pub publish` and
-creating and pushing the tag in git if there are issues with it.
-
-Install the tool by running:
-
-```terminal
-$ pub global activate flutter_plugin_tools
-```
-
-Then, from the root of your local `flutter/plugins` repo, use the tool to
-publish a release.
-
-```terminal
-$ pub global run flutter_plugin_tools publish-plugin --package $package
-```
-
-By default the tool tries to push tags to the `upstream` remote, but that and
-some additional settings can be configured. Run `pub global activate
-flutter_plugin_tools --help` for more usage information.
-
-The tool wraps `pub publish` for pushing the package to pub, and then will
-automatically use git to try and create and push tags. It has some additional
-safety checking around `pub publish` too. By default `pub publish` publishes
-_everything_, including untracked or uncommitted files in version control.
-`flutter_plugin_tools publish-plugin` will first check the status of the local
-directory and refuse to publish if there are any mismatched files with version
-control present.
-
-There is a lot about this process that is still to be desired. Some top level
-items are being tracked in
-[flutter/flutter#27258](https://github.com/flutter/flutter/issues/27258).
+For convenience, a [publish-plugin](./script/tool/README.MD#publish-and-tag-release) tool script could be used to do both step at once.
```

Comment:
Let's flip this section to say to use this script, and then give the manual steps as fallback. We should lead with the option we want people to use, and this is no longer experimental given that we use it all the time.